.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: krew-index CI
+
+on:
+- push
+- pull_request
+
+jobs:
+  build:
+    name: Build and Tags
+    runs-on: ubuntu-latest
+    env: 
+      KREW_VERSION: v0.3.3
+      GO111MODULE: on
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Install kubectl
+      run: |
+        set -euo pipefail
+        sudo curl -fsSL -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.13.1/bin/linux/amd64/kubectl
+        sudo chmod +x /usr/bin/kubectl        
+
+    - name: Install krew
+      shell: bash
```

Comment:
as it's the default, let's remove `shell: bash` here.

Reply:
Sure, it's done.